### PR TITLE
Add support for PHP8

### PR DESCRIPTION
**Pull request by ben221199**
This PR adds support for PHP8.

### CODE REVIEW

**Comment by koenhoeijmakers** [comment]
I think this has a better chance of being merged if you'd just change the php requirement to `"php": ">=7.1",` without all the unnecessary reformatting.

Also could you supply a link to the documentation that states that the `$refCode` should be a `string` or `null`?

**Comment by ben221199** [comment]
If you look at the documentation (https://ekomi01.atlassian.net/wiki/spaces/KB/pages/3131146248/KV+Kiyoh+Invite+API), the `refCode` also allows non-decimal characters.

**Review by ricardovanlaarhoven — CHANGES_REQUESTED** [review]
Please don't change the formatting and for example the indentation into tabs instead of spaces.
This also makes the PR hard to read.

Could you revert those changes and only leave your functional changes

File: src/API/GetReviews.php [hunk 1]
```diff
@@ -123,5 +123,4 @@ private function mapIntoModels(array $data): ProfileModel
 
         return new ProfileModel($data, $reviewModels);
     }
-}
-
+}
```
Comment (ricardovanlaarhoven) [inline src/API/GetReviews.php:126]: There are still some formatting changes

File: composer.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": ">=7.1",
```
Comment (ricardovanlaarhoven) [inline composer.json:14]: Wasn't php8 added here before? Are you sure you want to remove this?
The current `>=7.1` seems good for php8 as well, right?

**Review by ricardovanlaarhoven — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (ben221199) [reply composer.json:14]: https://github.com/kingscode/php-klanten-vertellen-api/pull/5#issuecomment-905565291

Comment (jspekken) [reply composer.json:14]: This also matches PHP 8.1 or 9 or... that might cause unnecessary problems. `^7.1|^8.0` is stricter and better suited for whatever the future brings in PHP land.

**Review by jspekken — CHANGES_REQUESTED** [review]

**Comment by ben221199** [comment]
🤔

**Pull request closed without merging**